redaxo/src/addons/cronjob/lib/types/phpcode.php

```diff
@@ -15,7 +15,16 @@ public function execute()
         $code = preg_replace('/^\<\?(?:php)?/', '', $this->getParam('code'));
         $is = ini_set('display_errors', true);
         ob_start();
-        $return = eval($code);
+        $return = false;
+
+        try {
+            $return = eval($code);
+        } catch (Throwable $exception) {
+            echo get_class($exception).': '.$exception->getMessage();
```

Comment:
wo landet die ausgabe? Im cron-log?

Reply:
Ja, in Zeile 28 wird der gesamte Output von eval, inkl. dieser Ausgaben abgefangen und in die Message verwandelt.